src/bridge/Makefile.am

```diff
@@ -42,7 +42,6 @@ libcockpit_bridge_a_SOURCES = \
 	src/bridge/cockpitdbusprocess.c \
 	src/bridge/cockpitdbusrules.c \
 	src/bridge/cockpitdbusrules.h \
-	src/bridge/cockpitdbussetup.c \
```

Comment:
I'm confused, why is this removed in this commit?

Reply:
That is the theme of the commit, replacing user synch with ssh setup. Do you want to keep the API, or do you want the removal to happen in a separate commit?

Reply:
Oh ic, nvm it's just me not knowing the bridge very well :)

Reply:
IMHO it's fine to remove this, but it means that an old dashboard and shell (machine-dialogs.js consumers) will fail against a new bridge. This should not commonly happen in practice, but it'd be better to make this explicit by adding `Conflicts:` in the spec and Breaks: in tools/debian/control.